generic_throttle/README.md

```diff
@@ -0,0 +1,20 @@
+# generic_throttle
+
+generic_throttle is package offering a Python implementation of a throttle for topics.
+
+Class GenericThrottle runs a ROS node which subscribes to a list of topics and
+publishes them again at a desired rate. List of topics and desired rate are
+specified through ROS parameters.
+
+By default, the throttle operates in a latched mode: if no new message are available
+from the input topics, it will continue to publish the last available message.
+However, after a certain time (to be defined in parameter /generic_throttle/delay
+in seconds) it will throw a warning and stop publishing the old message. As soon as
+new messages become available, the throttle will start to publish again.
```

Comment:
Please update the README and describe an example yaml layout, i.e. mandatory and optional parameters